README.md

```diff
@@ -16,7 +17,13 @@ This is my first time coding in Rust so fixes and improvements are appreciated :
 ```zsh
 # OSD window
 exec swayosd
+```
+or
```

Comment:
"or start with a max volume set" or something similar

Reply:
Done. Also added what the default is.